### PR TITLE
[US400102 US400103 US400104] Enhance frontend logic with order 

### PR DESCRIPTION
**Pull request by hieuhoang2901**

### CODE REVIEW

File: apps/backend/order-ms/src/main/resources/application.properties
Comment (duyphatbk) [inline apps/backend/order-ms/src/main/resources/application.properties:1]: discard change

File: apps/backend/order-ms/src/main/java/com/service/order/utils/ExceptionHandlerUtil.java
Comment (duyphatbk) [inline apps/backend/order-ms/src/main/java/com/service/order/utils/ExceptionHandlerUtil.java:1]: discard change

File: apps/backend/order-ms/src/main/java/com/service/order/security/OrderServiceInterceptorAppConfig.java
Comment (duyphatbk) [inline apps/backend/order-ms/src/main/java/com/service/order/security/OrderServiceInterceptorAppConfig.java:1]: discard change

File: apps/backend/order-ms/src/main/java/com/service/order/controllers/OrderController.java
Comment (duyphatbk) [inline apps/backend/order-ms/src/main/java/com/service/order/controllers/OrderController.java:1]: discard change

**Review by duyphatbk — APPROVED** [review]

**Review by LittleLegend-Pengiun — APPROVED** [review]

**Pull request merged**